tf_encrypted/operations/secure_random/secure_random.py

```diff
@@ -0,0 +1,47 @@
+import tensorflow as tf
+import tf_encrypted as tfe
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import dtypes
+import os
+
+dirname = os.path.dirname(tfe.__file__)
+shared_object = dirname + '/operations/secure_random/secure_random_module.so'
+
+
+def secure_random(shape, minval=0, maxval=None, dtype=tf.int32, seed=None, name=None):
```

Comment:
I suggest we have three operations actually:
- one to generate seeds
- one to expand seeds into full tensors
- one to securely sample full tensors

the first two are strongly correlated, the third is unrelated (at least on the surface; it may use whatever underneath)

Reply:
I was thinking of deferring this sort of thing to other PRs. What do you think?